app.js

```diff
@@ -71,19 +110,35 @@ var postingStrategies = {
   file: postFile
 };
 
-bot.onPinAdded(function (message, channelData) {
+var tweacji = (env.REACJI_TO_TRIGGER_TWEET || 'pushpin').split(',');
+var tweacjiFormatted = tweacji.map(function (ji) { return `:${ji}:`; });
+var tweacjiList = tweacjiFormatted.pop();
+if (tweacjiFormatted.length > 0) {
+  tweacjiList = tweacjiFormatted.join(', ') + ' or ' + tweacjiList;
+}
+
+bot.onItemPinned(function (event, channelData) {
+  var thing = 'something';
+  if (event.item && event.item.message && event.item.message.permalink) {
+    thing = event.item.message.permalink;
+  }
+  var text = `You just pinned ${thing} in #${channelData.channel.name}. Were you trying to get the Pinwheel bot to tweet it? *We've switched to using reacji instead of pins*, because pins are annoying and limited. So go react with ${tweacjiList} instead!`;
```

Comment:
Are we planning to keep this forever? Seems like it will get annoying. We should let the user message the bot to acknowledge that they've read the message and then they won't see it again. We can do this in a separate PR (although we should be ready to push a simple PR to remove the DM if people are annoyed by it).

Reply:
Yeah, we definitely want to do that. I actually want to enhance the message with Slack "message buttons", so the user could click one button to unpin-and-reacji instead. But we'd need to have a public callback URL that we could construct from environment variables, like `https://some-wacky-EC2.domain/unpin-and-react/id`.